docs/edit-layer.md

```diff
@@ -55,6 +56,12 @@ From the DataBC Details panel, you can modify the following attributes:
 
 `Display Range` allows you to specify an allowable range where this layer can be displayed. If your map is outside of this scale range, the layer will not draw on the map, and will not be identifiable or selectable.
 
+#### Show Labels
+
+`Show labels` appears when a DataBC layer includes configuration for the display of labels. Toggling this value will turn labels on or off.
+
+Modifications can be made to how labels are displayed by manually editing `smk-config.json`. Each DataBC layer in the `layers` property has a `dynamicLayers` property. This includes a `drawingInfo` property which defines the appearance of the layer on the map. Label content and style can be updated by editing its `labelingInfo` property. For more information, see <a href="https://developers.arcgis.com/web-map-specification/objects/drawingInfo">`drawingInfo` documentation</a>.
```

Comment:
We could consider expanding on what differentiates a DataBC layer from the other layer types in case someone has added many layers to their app. Just a thought, not sure if it is necessary.

Each DataBC layer has an array of `dynamicLayers`. A DataBC layer can be identified by its type which is `esri-dynamic`. Each item in the `dynamicLayers` array has a `drawingInfo` property ..."

Reply:
Thanks for the suggestion, but I think I'll leave it out as the dynamicLayers property only appears for DataBC layers.